### DISCUSSION TITLE
Can I make it hotswap?

### DISCUSSION THREAD

**Discussion by Fidexy** [ZeroZeroOne-dev/00Key#15] [General]
Is it possible to add hotswap capability with Mill-max 7305 sockets?

**Comment by ZeroZeroOne-dev** [comment]
I haven't tried it but I don't see why not.

Make sure to use a plate, or at least be sure to use 5-pin switches so they are a bit more stable.

**Reply by Fidexy** [reply]
Thanks for the advice! My build is almost complete and I just need to put the case, plate and PCB together. However, I don't really get how the PCB is mounted to the plate. Do I put a spacer between the plate and PCB? If so, what length should I use?

**Reply by ZeroZeroOne-dev** [reply]
Ah, right, normally they aren't connected because the soldered in switches provide the support between the two. but for hotswap that is less the case. Still I think you should try just not having the plate connected to the pcb and see if the mill-max sockets provide enough hold to keep the pcb up.

**Reply by ZeroZeroOne-dev** [reply]
If this doesn't work, there are holes in the plate and pcb that line up where you could install some spacers. You'll have to test out some lengths but i think 4mm spacers (if you take into account the extra height added by the mill-max sockets) for a 1.5 mm plate should be ok.
